src/netbase.cpp

```diff
@@ -68,6 +68,20 @@ std::string GetNetworkName(enum Network net)
     assert(false);
 }
 
+std::vector<std::string> GetNetworkNames(bool append_unroutable)
+{
+    std::vector<std::string> names;
+    for (int n = 0; n < NET_MAX; ++n) {
+        const enum Network network{static_cast<Network>(n)};
+        if (network == NET_UNROUTABLE || network == NET_I2P || network == NET_CJDNS || network == NET_INTERNAL) continue;
```

Comment:
@vasild you may want to update this line (and the feature_proxy tests) in #20685 if you're inclined to it in that pull